### ISSUE TITLE
For bootstrapping on-prem-builder installs it would be nice to automate the creation of the core origin

### ISSUE THREAD

**Issue by srb3** [habitat-sh/on-prem-builder#118]
The core origin is needed so you complete the bootstrapping of core packages (https://github.com/habitat-sh/on-prem-builder#bootstrap-core-packages).
This is currently a manual step, with no clear way to do this from the api.

**Comment by bdausses** [comment]
Agree.  Would be nice to see some automated way to do this... and optionally to start a core package sync.  As is, this is a roadblock to getting the builder setup in an automated fashion.
